core/server/data/migrations/versions/3.26/02-refactor-slack-setting.js

```diff
@@ -0,0 +1,95 @@
+module.exports = {
+    config: {
+        transaction: true
+    },
+
+    async up({transacting: knex}) {
+        const slackSetting = await knex('settings')
+            .select('value')
+            .where({
+                key: 'slack'
+            })
+            .first();
+
+        let slackUrl = '';
+        let slackUsername = '';
+        try {
+            const value = JSON.parse(slackSetting.value);
+
+            slackUrl = value.url;
+            slackUsername = value.username;
+        } catch (err) {
+            slackUrl = '';
+            slackUsername = 'Ghost';
```

Comment:
@allouis can't find any reference to why a default username was chosen to be `'Ghost'` instead of leaving it as an empty string `''`?

Reply:
It's following the default from here: https://github.com/TryGhost/Ghost/blob/main/core/server/data/schema/default-settings.json#L405

Reply:
Oh right, thanks! :pray: